habitat/utils/pickle5_multiprocessing.py

```diff
@@ -75,3 +76,25 @@ def __getattr__(self, name):
                 type(self).__name__, "conn"
             )
         )
+
+
+class CloudpickleWrapper:
+    """Wrapper that uses cloudpickle to pickle and unpickle the result."""
+
+    def __init__(self, fn: Callable):
+        """Cloudpickle wrapper for a function."""
+        self.fn = fn
+
+    def __getstate__(self):
+        """Get the state using `cloudpickle.dumps(self.fn)`."""
+        import cloudpickle
+
+        return cloudpickle.dumps(self.fn)
+
+    def __setstate__(self, ob):
+        """Sets the state with obs."""
+        self.fn = pickle.loads(ob)
```

Comment:
This code was copied from https://github.com/openai/gym/blob/94081fbc0ade19747c9eea8aa435310f46971873/gym/vector/utils/misc.py#L8

I need to copy it because I am changing the signature of `__call__`.
I tested the functionality and it seems to work.